src/lib/backend.ai-client-esm.ts

```diff
@@ -2827,7 +2827,7 @@ class ContainerImage {
     if (Object.keys(resource).length === 0) {
       resource = {'cpu': '1', 'mem': '512m'};
     }
-    return this.client.createIfNotExists(registry + name, sessionId, resource, 600000, architecture).then((response) => {
+    return this.client.createIfNotExists(registry + name, sessionId, resource, Infinity, architecture).then((response) => {
       return this.client.destroy(sessionId);
```

Comment:
@Sujin-Kim1 Does it really "treat" timeout as "Infinity" in `createIfNotExists`?
`createIfNotExists` calls [`wrapWithPromise`](https://github.com/lablup/backend.ai-webui/blob/07f27fc1974abc25fa2adfdb28644c5e9d651803/src/lib/backend.ai-client-esm.ts#L951), and It inside the `wrapWithPromise` eventually calls `setTimeout` which doesn't treat `Infinity` as is.
Could you check it again?